data/YZA.yaml

```
@@ -0,0 +1,26 @@
+description: |-
+  Properties for yttria-stabilzed zirconia alumina (YZA) used in
+  porous media burners. Data compiled by Thorsten zirwes and used in
+  https://doi.org/10.1016/j.combustflame.2023.112642
+
+phases:
+- name: yttria_stabilzed_zirconia
```

Besides the spelling, we generally use dashes instead of underscores in the YAML format
```suggestion
- name: yttria-stabilized-zirconia
```